components/selection/test/selection.test.js

```diff
@@ -123,7 +123,6 @@ describe('d2l-selection-input', () => {
 				<d2l-selection-input key="key3" label="label3" disabled></d2l-selection-input>
 			</d2l-test-selection>
 		`);
-		await el.updateComplete;
 		await nextFrame();
```

Comment:
I wonder if this `nextFrame()` is still needed? I'm imagining you already tried to remove it...?

Reply:
There's a lot of these in here so I didn't want to mess with too much that I wasn't completely confident would be covered by the new `fixture`.